ast.go

```diff
@@ -592,6 +592,10 @@ type CreateDatabaseStatement struct {
 
 	// RetentionPolicyShardGroupDuration indicates shard group duration for the new database.
 	RetentionPolicyShardGroupDuration time.Duration
+
+	FutureWriteLimit *time.Duration
```

Comment:
They are pointers in `CreateDatabaseStatement` to allow the meta client to change defaults and determine whether a user set the defaults explicitly (and should not be overridden), or the meta client set the defaults.

> commit a6e7624cfec54bb4acfb1f3f0dbaee345b3ded0a
> Author: Jonathan A. Sternberg <jonathan@influxdb.com>
> Date:   Tue Aug 9 12:00:04 2016 -0500
> 
>     Use defaults from `meta` package for `CREATE DATABASE`
>     
>     Instead of having the parser set the defaults, the command will set the
>     defaults so that the constants for that are actually used. This way we
>     can also identify which things the user provided and which ones we are
>     filling with default values.
>     
>     This allows the meta client to be able to make smarter decisions when
>     determining if the user requested a conflict or if the requested
>     capabilities match with what is currently available. If you just say
>     `CREATE DATABASE WITH NAME myrp`, the user doesn't really care what the
>     duration of the retention policy is and just wants to use the default.
>     Now, we can use that information to determine if an existing retention
>     policy would conflict with what the user requested rather than returning
>     an error if a default value ever gets changed since the meta client
>     command can communicate intent more easily.
>